docs/developer-guide/unit-testing.md

```diff
@@ -384,19 +345,42 @@ A useful read about this topic can be found in the following article: https://st
 
 Mocking a service client using the following code ...
 
+Since we are mocking the provider, it can be customized setting multiple attributes to the provider:
```

Comment:
I think we need to place this in a different location in the document, please check the rendered view.

Reply:
Yes, I moved it to the `Mocking more than one service` section